rtgui/threadutils.h

```diff
@@ -59,7 +59,7 @@ class MyMutex :
 
 private:
     bool locked;
-    void checkLock ();
+    bool checkLock (bool noError=false);
```

Comment:
@Lawrence37 Good catch (as always)! :+1:

I'd rather read that with a bit more of whitespace: `bool noError = false`. :slightly_smiling_face: